sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestAdministrationClientTest.cs

```diff
@@ -0,0 +1,300 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Core.TestFramework.Models;
+using Azure.Developer.LoadTesting.Tests.Helper;
+using NUnit.Framework;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    public class LoadTestAdministrationClientTest: RecordedTestBase<LoadTestingClientTestEnvironment>
+    {
+        private string _testId;
+        private string _fileName;
+        private string resourceId;
+        private TestHelper _testHelper;
+
+        public LoadTestAdministrationClientTest(bool isAsync) : base(isAsync) {
+            _testId = "test-from-csharp-sdk-testing-framework";
+            _fileName = "sample.jmx";
+
+            _testHelper = new TestHelper();
+
+            BodyRegexSanitizers.Add(new BodyRegexSanitizer(@"sig=(?<group>.*?)(?=\s+)", SanitizeValue)
+            {
+                GroupForReplace = "group"
+            });
+        }
+
+        /* please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/tests/TemplateClientLiveTests.cs to write tests. */
+        private LoadTestAdministrationClient CreateAdministrationClient()
+        {
+            return InstrumentClient(new LoadTestAdministrationClient(new Uri("https://" + TestEnvironment.Endpoint), TestEnvironment.Credential, InstrumentClientOptions(new LoadTestingClientOptions())));
+        }
+
+        [Test]
+        public async Task CreateOrUpdateTest()
+        {
+            LoadTestAdministrationClient loadTestAdministrationClient = CreateAdministrationClient();
```

Comment:
Instead of doing this in each test method, you could do it once in a setup method marked with the `[SetUp]` attribute. If there is nothing unique about it per-test, you could even just do it in the constructor.

Reply:
adding this with new tests